dist/tcti-tabrmd.pc.in

```diff
@@ -1,7 +1,7 @@
 Name: tcti-tabrmd
 Description: TCTI library for communicating with the TPM2 access broker / resource manager daemon (tabrmd).
-URL: https://github.com/01org/??
+URL: https://github.com/01org/tpm2-abrmd
 Version: @VERSION@
-Requires: sapi
+Requires: sapi glib-2.0 gio-2.0
```

Comment:
This is true. The library also relies on glib-2.0 as well though so I included both. I guess it is redundant though. Think it's worth removing?